src/woo_publications/metadata/theme_sync.py

```diff
@@ -0,0 +1,84 @@
+from io import StringIO
+
+from django.core.management import call_command
+
+import requests
+from glom import Coalesce, Path, PathAccessError, T, glom
+
+from woo_publications.metadata.models import Theme
+
+WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/scw_toplijst/1/json/scw_toplijst_1.json"
+
+SPEC = {
+    "data": {
+        "identifier": "@id",
+        "naam": T["http://www.w3.org/2004/02/skos/core#prefLabel"][0]["@value"],
+    },
+    "parents": Coalesce(
+        (T["http://www.w3.org/2004/02/skos/core#broader"], ["@id"]), default=[]
+    ),
+}
+
+
+class ThemeWaardenlijstError(Exception):
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(message)
+
+
+def update_theme(file_path: Path):
```

Comment:
`file_path` here is annotated as a `glom.Path`, not a `pathlib.Path` which is causing you type errors.

Removing `from glom import Path` and replacing it with `from pathlib import Path` should fix all issues

Reply:
Yes just noticed it myself...